test/integration/com/redhat/theses/UniversityControllerIntegrationSpec.groovy

```diff
@@ -0,0 +1,131 @@
+package com.redhat.theses
+
+import com.redhat.theses.auth.User
+import com.redhat.theses.auth.Role
+import org.codehaus.groovy.grails.plugins.springsecurity.SpringSecurityUtils
+import grails.plugin.spock.IntegrationSpec
+import spock.lang.Shared
+import spock.lang.Unroll
+import grails.test.mixin.*
+
+class UniversityControllerIntegrationSpec extends IntegrationSpec {
+    @Shared controller = new UniversityController()
+
+    def setupSpec() {
+    }
+
+    def cleanup() {
+    }
+
+    void "test show() true"() {
+        when: "show is called"
+        def model = controller.show(1)
+        then: "returned instance shouldn't be null"
+        model.universityInstance != null
+        model.universityInstance.name == "First"
+        model.universityInstance.acronym == "F"
+    }
```

Comment:
* Names such as "test show() true" are not really suitable names.
* Names such as "can retrieve university by id" are
* Empty lines in between test cases/methods would help improve the overall readability of test files. 

Compare your version with my, slightly modified one:
```groovy
void "can retrieve university"()  {
    when: "retrieving university by used id"
        def model = controller.show(1)
    then: "should retrieve the correct entity "
        model?.universityInstance?.name == "First"
        model?.universityInstance?.acronym == "F"
 }
```

BTW: ?. is a handy "safe" method call. Attribute/method will be called if it is defined on given object, otherwise null will be returned.